scripts/wrappers/dashboard_proxy.py

```diff
@@ -1,8 +1,14 @@
 #!/usr/bin/python3
 from subprocess import check_output
+import click
 
 
+@click.command(context_settings=dict(help_option_names=["-h", "--help"]))
```

Comment:
nit; we use `{}` instead of `dict`.

```suggestion
@click.command(context_settings={"help_option_names": ["-h", "--help"]})
```

Also in other places below.